src/DssSpell.sol

```diff
@@ -48,134 +57,134 @@ contract DssSpellAction is DssAction {
-
-        // WBTC-B: Decrease by 1 percentage point, from 10.25% to 9.25%
-        DssExecLib.setIlkStabilityFee("WBTC-B", NINE_PT_TWO_FIVE_PCT_RATE, /* doDrip = */ true);
-
-        // WBTC-C: Decrease by 1 percentage point, from 9.5% to 8.5%
-        DssExecLib.setIlkStabilityFee("WBTC-C", EIGHT_PT_FIVE_PCT_RATE, /* doDrip = */ true);
-
-        // Dai Savings Rate:
-        // Note: only heading, changes follow
-
-        // DSR: Decrease by 1 percentage point, from 8% to 7%
-        DssExecLib.setDSR(SEVEN_PCT_RATE, /* doDrip = */ true);
-
-        // ---------- Delegate Compensation ----------
-        // Forum: https://forum.makerdao.com/t/june-2024-aligned-delegate-compensation/24617
-        // MIP: https://mips.makerdao.com/mips/details/MIP101#2-6-aligned-delegates-ads-gov6
-
-        // BLUE - 41.67 MKR - 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf
-        MKR.transfer(BLUE, 41.67 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // JuliaChang - 41.67 MKR - 0x252abAEe2F4f4b8D39E5F12b163eDFb7fac7AED7
-        MKR.transfer(JULIACHANG, 41.67 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Byteron - 38.98 MKR - 0xc2982e72D060cab2387Dba96b846acb8c96EfF66
-        MKR.transfer(BYTERON, 38.98 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Cloaky - 20.40 MKR - 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818
-        MKR.transfer(CLOAKY, 20.40 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // PBG - 16.58 MKR - 0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2
-        MKR.transfer(PBG, 16.58 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // BONAPUBLICA - 13.89 MKR - 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3
-        MKR.transfer(BONAPUBLICA, 13.89 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Rocky - 13.89 MKR - 0xC31637BDA32a0811E39456A59022D2C386cb2C85
-        MKR.transfer(ROCKY, 13.89 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // vigilant - 12.55 MKR - 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61
-        MKR.transfer(VIGILANT, 12.55 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Ennoia (Cloaky) - 4.10 MKR - 0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b
-        MKR.transfer(CLOAKY_ENNOIA, 4.10 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // Kohla (Cloaky) - 4.10 MKR - 0xA9D43465B43ab95050140668c87A2106C73CA811
-        MKR.transfer(CLOAKY_KOHLA, 4.10 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
-
-        // WBC - 1.34 MKR - 0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47
-        MKR.transfer(WBC, 1.34 ether); // Note: 'ether' is a keyword helper, only MKR is transferred here
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
```

Comment:
- `dstIlk` is actually listed in the section header.
- `srcPsmKey` is more of a technical parameter. The ilk `PSM-USDC-A` is listed in one of the section headers, and `MCD_PSM_USDC_A` is its chainlog key counterpart. I'm going to add a note.
- `dstPip` and `dstGem` are sort of implicit because we are clearly dealing with USDC components. I'm adding a note because I don't believe having that listed in the exec sheet is absolutely mandatory.